dev/RadialGradientBrush/RadialGradientBrush.cpp

```diff
@@ -197,14 +197,16 @@ void RadialGradientBrush::UpdateCompositionGradientMappingMode()
     }
 }
 
-void RadialGradientBrush::UpdateCompositionGradientOriginOffset()
+void RadialGradientBrush::UpdateCompositionGradientOrigin()
 {
     MUX_ASSERT(SharedHelpers::IsCompositionRadialGradientBrushAvailable());
 
     if (const auto compositionGradientBrush = m_brush.try_as<winrt::CompositionRadialGradientBrush>())
     {
-        const auto gradientOriginOffset = GradientOriginOffset();
-        compositionGradientBrush.GradientOriginOffset(winrt::float2(gradientOriginOffset.X, gradientOriginOffset.Y));
+        const auto gradientOrigin = GradientOrigin();
+        // This sets the gradient offset center to the top left corner
+        // Top Left is (-0.5,-0.5), center is (0,0)
```

Comment:
Can you please double check this with the behavior and ensure that the default and custom values create the same visuals as WPF ?

Reply:
Default value is (.5, .5) in WPF.
Here is a comparison of WinUI (left) and WPF (right)
From top to bottom: GradientOrigin (.5,.5), (0,0) and (1,1):

![image](https://user-images.githubusercontent.com/16122379/77848978-a7c00a80-71c8-11ea-9928-44124a140aaa.png)